.gitignore

```diff
@@ -34,3 +34,5 @@ node_modules
 # IntelliJ
 .idea
 *.iml
+
+go.sum
```

Comment:
I'm more in favor of keeping `go.sum` as described here: https://github.com/golang/go/wiki/Modules#should-i-commit-my-gosum-file-as-well-as-my-gomod-file